backend/conferences/admin.py

```diff
@@ -102,6 +102,10 @@ class ConferenceAdmin(admin.ModelAdmin):
             "Pretix",
             {"fields": ("pretix_organizer_id", "pretix_event_id", "pretix_event_url")},
         ),
+        (
+            "Speaker Voucher",
+            {"fields": ("pretix_speaker_voucher_quota_id",)},
```

Comment:
Just curious: why another pretix id?

Reply:
this is the ID of the quota to use to assign to the vouchers code